src/sweetAll.ttl

```diff
@@ -230,6 +230,9 @@
                                                  <http://sweetontology.net/stateTimeGeologic> ,
                                                  <http://sweetontology.net/stateVisibility> ;
                                      rdfs:label "SWEET Ontology" ;
+									 	rdfs:comment """The semantic web for Earth and environmental terminology (SWEET) is an investigation in improving discovery and use of Earth science data, through software understanding of the semantics of web resources.  SWEET is a collection of ontologies conceptualizing a knowledge space for Earth system science, represented using the web ontology language (OWL). It includes both orthogonal concepts (space, time, Earth realms, physical quantities, etc.) and integrative science knowledge concepts (phenomena, events, etc.).  The ontology is modularized into 224 component files, each representing a different namespace, with numerous dependencies implemented by OWL imports between the namespace.
```

Comment:
@smrgeoinfo I would be tempted not to specify the number of component files as what happens if we add one more, take one away or produce more alignments...
The rest of this description is excellent.

Reply:
This brings up some deep questions about versioning-- right now v3.2.0 has 225 files-- one master, and 224 components.  I assume part of the logic for the components is that they could be versioned separately by additions, updates, deletions, or refactoring between components.   I assume if some component versions we updated then a new master version would be required that imported a different set of component versions.   In this model, the number of components in v3.2.0 master will always be 224.